hicommonwealth/commonwealth
# add TriggerNotificationsWorkflow command

## Link to Issue
Closes: #8699 

## Description of Changes
- Adds a super admin route to trigger an arbitrary Knock workflow
- Adds form on super admin panel for calling the new route

## Test Plan
- Delete your account with `pnpm delete-user [id or email]`
- Create a new user account
- Set super admin with `pnpm set-super-admin`
- Go to the admin panel
- Set the key: `all-user-test`
- Set data to a JSON object with 2 keys: 'some_value' and 'url'
- Hit Trigger and you should receive a notification with `some_value` in the body.

## Deployment Plan

1. 

## Other Considerations

-
